### PR TITLE
Fixed wazuh-passwords-tool.sh so it doesn't remove the password in kibana.yml [stable]

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-packages/issues/920|

## Description

I have changed wazuh-passwords-tool.sh so that it only changes the kibana.yml elasticsearch password if the user or one of the users changed is kibanaserver 

## Logs example

kibana_passwords
>

## Tests

Checked on CentOS 7 and Ubuntu 18

### CODE REVIEW

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh [hunk 1]
```diff
@@ -168,35 +168,40 @@ User:
     fi
 
     if [ -n "${CHANGEALL}" ]; then
-        for i in "${!USERS[@]}"; do
```
Comment (verdx) [inline unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh:171]: The changes in these lines correspond to a fix for https://github.com/wazuh/wazuh-packages/pull/937 that slipped through

**Comment by verdx** [comment]
It has been reviewed and tested and it works perfectly.
```
[root@centos7 ~]# cat /etc/kibana/kibana.yml 
server.host: 0.0.0.0
elasticsearch.hosts: https://127.0.0.1:9200
server.port: 443
elasticsearch.ssl.verificationMode: certificate
elasticsearch.username: kibanaserver
elasticsearch.password: kibanaserver2
elasticsearch.requestHeadersWhitelist: ["securitytenant","Authorization"]
opendistro_security.multitenancy.enabled: true
opendistro_security.readonly_mode.roles: ["kibana_read_only"]
server.ssl.enabled: true
server.ssl.key: "/etc/kibana/certs/kibana-key.pem"
server.ssl.certificate: "/etc/kibana/certs/kibana.pem"
elasticsearch.ssl.certificateAuthorities: ["/etc/kibana/certs/root-ca.pem"]
server.defaultRoute: /app/wazuh?security_tenant=global
[root@centos7 ~]# cat /etc/kibana/kibana.yml 
server.host: 0.0.0.0
elasticsearch.hosts: https://127.0.0.1:9200
server.port: 443
elasticsearch.ssl.verificationMode: certificate
elasticsearch.username: kibanaserver
elasticsearch.password: kibanaserver2
elasticsearch.requestHeadersWhitelist: ["securitytenant","Authorization"]
opendistro_security.multitenancy.enabled: true
opendistro_security.readonly_mode.roles: ["kibana_read_only"]
server.ssl.enabled: true
server.ssl.key: "/etc/kibana/certs/kibana-key.pem"
server.ssl.certificate: "/etc/kibana/certs/kibana.pem"
elasticsearch.ssl.certificateAuthorities: ["/etc/kibana/certs/root-ca.pem"]
server.defaultRoute: /app/wazuh?security_tenant=global
[root@centos7 ~]# bash /vagrant/wazuh-packages/unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh  -u wazuh
Generating random password
Done
Creating backup...
Backup created
Generating hash
Hash generated
Filebeat started
Loading changes...
Done

The password for user wazuh is kpUQADN_rW0PcuaXGagKfcbnAmJSrLO_
Password changed. Remember to update the password in /etc/filebeat/filebeat.yml and /etc/kibana/kibana.yml if necessary and restart the services.
[root@centos7 ~]# cat /etc/kibana/kibana.yml 
server.host: 0.0.0.0
elasticsearch.hosts: https://127.0.0.1:9200
server.port: 443
elasticsearch.ssl.verificationMode: certificate
elasticsearch.username: kibanaserver
elasticsearch.password: kibanaserver2
elasticsearch.requestHeadersWhitelist: ["securitytenant","Authorization"]
opendistro_security.multitenancy.enabled: true
opendistro_security.readonly_mode.roles: ["kibana_read_only"]
server.ssl.enabled: true
server.ssl.key: "/etc/kibana/certs/kibana-key.pem"
server.ssl.certificate: "/etc/kibana/certs/kibana.pem"
elasticsearch.ssl.certificateAuthorities: ["/etc/kibana/certs/root-ca.pem"]
server.defaultRoute: /app/wazuh?security_tenant=global
[root@centos7 ~]# 

```

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh [hunk 2]
```diff
@@ -168,35 +168,40 @@ User:
...
+		        fi
+	        done
+        if [ $supported = false ]; then
+	        echo "Error: The given user ${FILEUSERS[j]} does not exist"
+	      fi
         done
     else
-	FINALUSERS=()
-	FINALPASSWORDS=()
+	    FINALUSERS=()
```
Comment (DFolchA) [inline unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh:184]: Please fix indents

**Review by DFolchA — CHANGES_REQUESTED** [review]

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh [hunk 3]
```diff
@@ -168,35 +168,40 @@ User:
     fi
 
     if [ -n "${CHANGEALL}" ]; then
-        for i in "${!USERS[@]}"; do
-	    for j in "${!FILEUSERS[@]}"; do
-	        if [[ ${USERS[i]} == ${FILEUSERS[j]} ]]; then
-		    PASSWORDS[i]=${FILEPASSWORDS[j]}
-		fi
-	    done
+        for j in "${!FILEUSERS[@]}"; do
```
Comment (verdx) [inline unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh:171]: These changes correspond to a fix related to https://github.com/wazuh/wazuh-packages/pull/937

**Review by DFolchA — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**